requirements.txt

```diff
@@ -1,7 +1,7 @@
 h5py>=2.10.0
 wheel>=0.33.1
 future>=0.18.2
-numpy>=1.18.5,<=1.19.2
+numpy>=1.18.5
```

Comment:
`tensorflow-gpu` requires 19.5, but there's no reason to limit that if they aren't installing `tensorflow-gpu`